packages/loader/container-loader/README.md

```diff
@@ -178,3 +181,11 @@ Hosts can also force readonly-mode for a container via calling `Container.forceR
 
 ### `storageOnly`
 Storage-only mode is a readonly mode in which the container does not connect to the delta stream and is unable to submit or recieve ops. This is useful for viewing a specific version of a document.
+
+## Dirty events
+Container runtime (IRuntime) communicates with Container about state of the container. In response, Container will raise two events - `"dirty"` and `"saved"` events. Transitions between twese two events signify presence (or lack of) user changes that were not saved to storage. In other words, if container is dirty, closing it at that moment will result in data loss from user perspective, as not all user changes made it to storage.
```

Comment:
```suggestion
The Container runtime can communicate with the container to get the container's current state. In response, the container will raise two events - `"dirty"` and `"saved"` events. Transitions between these two events signify presence (or lack of) user changes that were not saved to storage. In other words, if container is dirty, closing it at that moment will result in data loss from user perspective, because not all user changes made it to storage.
```